tests/parse/test_parse_case.py

```diff
@@ -54,30 +54,39 @@ def test_parse_case_date(scout_config):
     assert isinstance(case_data["analysis_date"], datetime)
 
 
-def test_parse_case_owner(scout_config):
-    # GIVEN you load sample information from a scout config
-    # WHEN case is parsed
-    case_data = parse_case(scout_config)
-    # THEN the case should have a owner
-    assert case_data["owner"] == scout_config["owner"]
-
-
-def test_parse_case_limsid(scout_config):
-    """Test parsing a case with lims_id"""
-
+@pytest.mark.parametrize(
```

Comment:
This is already much better than having a lot of single tests! 🥇 

Since you are touching it you could add the missing keys that maintain the same names when the case is parsed: cohorts, phenotype_terms, gene_fusion_report, sv_rank_model_version, peddy_check, peddy_sex

Reply:
Done!